bin/phpcrsh

```diff
@@ -1,15 +1,16 @@
 #!/usr/bin/env php
 <?php
 
-$vendorDir = __DIR__.'/../vendor';
-
-$file = $vendorDir.'/autoload.php';
-
-if (file_exists($file)) {
-    $autoload = require_once $file;
-} else {
-    echo 'Cannot find the vendor directory, have you executed composer install?';
-    exit(1);
+if (!class_exists('\PHPCR\Shell\PhpcrShell')) {
+    if (is_file(__DIR__.'/../vendor/autoload.php')) {
+        require __DIR__.'/../vendor/autoload.php';
+    } elseif (is_file(__DIR__.'/../../../autoload.php')) {
```

Comment:
Yes. This is case of symlink to binary from ``COMPOSER_BIN_DIR (default vendor/composer/bin)``  to ``vendor/phpcr/phpcr-shell/bin``.